src/miner.cpp

```diff
@@ -473,6 +473,9 @@ void static BitcoinMiner(CWallet *pwallet)
     );
 
     try {
+        unique_ptr<equi> peq;
+        if (solver == "tromp")
+            peq.reset(new equi(1));
```

Comment:
To build with devtoolset-3's g++ 4.9.1, I changed these 3 lines to:

```
        equi *peq;
        if (solver == "tromp")
            peq = new equi(1);
```

It built and seems to work - memory allocation is now stable - but somehow I am not seeing much speedup.